2021-10-28 21:51:06,635 INFO    MainThread:21569 [wandb_setup.py:_flush():71] setting env: {}
2021-10-28 21:51:06,635 INFO    MainThread:21569 [wandb_init.py:_log_setup():357] Logging user logs to /home/siddhi/workspace/ColorCapsNet/wandb/offline-run-20211028_215106-1nlruu4e/logs/debug.log
2021-10-28 21:51:06,635 INFO    MainThread:21569 [wandb_init.py:_log_setup():358] Logging internal logs to /home/siddhi/workspace/ColorCapsNet/wandb/offline-run-20211028_215106-1nlruu4e/logs/debug-internal.log
2021-10-28 21:51:06,636 INFO    MainThread:21569 [wandb_init.py:init():390] calling init triggers
2021-10-28 21:51:06,636 INFO    MainThread:21569 [wandb_init.py:init():395] wandb.init called with sweep_config: {}
config: {}
2021-10-28 21:51:06,636 INFO    MainThread:21569 [wandb_init.py:init():435] starting backend
2021-10-28 21:51:06,636 INFO    MainThread:21569 [backend.py:_multiprocessing_setup():94] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-28 21:51:06,695 INFO    MainThread:21569 [backend.py:ensure_launched():198] starting backend process...
2021-10-28 21:51:06,734 INFO    MainThread:21569 [backend.py:ensure_launched():203] started backend process with pid: 21614
2021-10-28 21:51:06,735 INFO    MainThread:21569 [wandb_init.py:init():444] backend started and connected
2021-10-28 21:51:06,737 INFO    MainThread:21569 [wandb_init.py:init():503] updated telemetry
2021-10-28 21:51:06,747 INFO    MainThread:21569 [wandb_init.py:init():576] starting run threads in backend
2021-10-28 21:51:11,748 INFO    MainThread:21569 [wandb_run.py:_console_start():1693] atexit reg
2021-10-28 21:51:11,749 INFO    MainThread:21569 [wandb_run.py:_redirect():1567] redirect: SettingsConsole.REDIRECT
2021-10-28 21:51:11,749 INFO    MainThread:21569 [wandb_run.py:_redirect():1572] Redirecting console.
2021-10-28 21:51:11,750 INFO    MainThread:21569 [wandb_run.py:_redirect():1628] Redirects installed.
2021-10-28 21:51:11,750 INFO    MainThread:21569 [wandb_init.py:init():603] run started, returning control to user process
2021-10-28 21:53:04,596 INFO    MainThread:21569 [wandb_run.py:_atexit_cleanup():1663] got exitcode: 1
2021-10-28 21:53:04,851 INFO    MainThread:21569 [wandb_run.py:_restore():1635] restore
2021-10-28 21:53:12,266 INFO    MainThread:21569 [wandb_run.py:_wait_for_finish():1793] got exit ret: 
2021-10-28 21:53:12,455 INFO    MainThread:21569 [wandb_run.py:_wait_for_finish():1793] got exit ret: 
2021-10-28 21:53:12,557 INFO    MainThread:21569 [wandb_run.py:_wait_for_finish():1793] got exit ret: 
2021-10-28 21:53:12,884 INFO    MainThread:21569 [wandb_run.py:_wait_for_finish():1793] got exit ret: done: true
exit_result {
}
local_info {
}

